### PR TITLE
[MVC-1105] Cleaned up code and fixed missing top menu 

### PR DESCRIPTION
**Pull request by nashthecoder**

### CODE REVIEW

**Comment by nashthecoder** [comment]
Screenshot 2021-04-22 at 11 50 15
Screenshot 2021-04-22 at 11 50 06

File: _layouts/post.html [hunk 1]
```diff
@@ -13,4 +13,4 @@ <h1 class="post-title" itemprop="name headline">{{ page.title }}</h1>
     {{ content }}
   </div>
 
-</article>
+</article> -->
```
Comment (raguiar9080) [inline _layouts/post.html:16]: Can't this whole file be removed since it's all commented now?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+    "terminal.integrated.fontFamily": "Hack Nerd Font"
+}
```
Comment (raguiar9080) [inline .vscode/settings.json]: `.vscode` should probably be added to the list on `.gitignore`. I'm assuming this is local visual studio configs so not directly related to the project.

**Review by raguiar9080 — COMMENTED** [review]
Is this supposed to replace PR https://github.com/Triggerise/triggerise.github.io/pull/11 or is it extra fixes to sit on top of it?
If it's the first case, I would close the old one and add all the necessary info to the PR description here (from what is here, one wouldn't expecte Policy changes).
If it's the second case, you need to change the destination branch to the one in that PR instead of develop.
Nevertheless, left some two small comments but looking good in general, things do seem cleaner 👍

File: .vscode/settings.json (hunk 2 above)
Comment (nashthecoder) [reply .vscode/settings.json]: On it...

File: _layouts/post.html (hunk 1 above)
Comment (nashthecoder) [reply _layouts/post.html:16]: Let me sort that.. Need to get the hang of this branch business and .gitignore.

**Pull request merged**